.github/workflows/comment-goodfirstissue.yml

```
@@ -0,0 +1,35 @@
+name: Comment on Good First Issue
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check if labeled as "good first issue"
+        id: check_label
+        run: |
+          echo ::set-output name=is_good_first_issue::${{ contains(github.event.issue.labels.*.name, 'good first issue') }}
+
+      - name: Add comment
+        if: steps.check_label.outputs.is_good_first_issue == 'true'
+        uses: actions/github-script@v6
```

```suggestion
  comment:
    if: github.event.label.name == 'good first issue'
    runs-on: ubuntu-latest
      - name: Add comment
        uses: actions/github-script@v6
```

Code here can be simplified, see https://docs.github.com/en/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added. 

Also noticed that this PR continues from #12480. Though most of the code will have been changed after this PR, it would still be good if you can edit the description to credit the original author.